api/grpcserver/globalstate_service.go

```diff
@@ -274,13 +283,37 @@ func (s GlobalStateService) AccountDataStream(in *pb.AccountDataStreamRequest, s
 		channelReceipt chan events.TxReceipt
 	)
 	if filterAccount {
-		channelAccount = events.GetAccountChannel()
+		if s.accountDataStreamAccountChannel == nil {
+			channelAccount = events.SubscribeToAccounts()
+			if channelAccount == nil {
+				log.Error("Reporter is not running")
```

Comment:
It's redundant and probably unnecessary to print the error here verbatim right before returning it. Make sure the caller properly handles/surfaces the error, then you can remove all of these `log.Error` statements.